modules/local/mark_duplicates.nf

```diff
@@ -11,7 +11,7 @@ process MARK_DUPLICATES {
      output:
          tuple idPatient, idSample, path("${idSample}.md.bam"), path("${idSample}.md.bam.bai"), emit:  bam_duplicates_marked
          tuple idPatient, idSample, emit: tsv_bam_duplicates_marked
-         path "${idSample}.bam.metrics", emit: duplicates_marked_report //is optional , applies when skip_qc is used(not implemented yet)
+         path "${idSample}.bam.metrics", optional : true, emit: duplicates_marked_report //is optional , applies when skip_qc is used(not implemented yet)
```

Comment:
Actually we will need to rename this option.

you can't actually skip markduplicates with skip_qc, but you actually don't keep the report.

Reply:
Ah, I see. So maybe skip_markduplicates_report and make it an extra parameter not included in the skip_qc list?